### PR TITLE
Remove documentation for OpenBMC under Manage Cluster

### PR DESCRIPTION
**Pull request by whowutwut**
The OpenBMC is a new management type in xCAT.   The idea is to keep the external changes to a minimum so there's no reason we need to create another section just for OpenBMC this early in the development cycle.  

The section was added in https://github.com/xcat2/xcat-core/pull/2882 and I want to remove it before we release 2.13.4

### CODE REVIEW

**Comment by whowutwut** [comment]
I plan to make one more change to review the content under the Advanced Hardware commands in another commit... then it will be ready for merge.  But I've created this pull request for review to begin.  Need to merge this by the end of the week.

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/management/basic/rcons.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    * For OpenPower, **OpenBMC** managed servers: ::
+ 
+        chdef -t node -o <noderange> cons=openbmc
+
+Open a console to ``compute1``: ::
+
+    rcons compute1
+
+**Note:** The keystroke ``ctrl+E C .`` will disconnect you from the console.
```
Comment (gurevichmark) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/management/basic/rcons.rst]: "E" and "C" should be lower case characters.

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/management/basic/rbeacon.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
+``rbeacon`` - Beacon Light 
+==========================
+
+See :doc:`rbeacon manpage </guides/admin-guides/references/man1/rbeacon.1>` for more information.
+
+
+Most enterprise level servers have LEDs on their front and/or rear panels, one of which is a beacon light.  If turned on, this light can help assist the system administrator locate one physical machine out of a large number of machines in a frame.
```
Comment (gurevichmark) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/management/basic/rbeacon.rst]: The second sentence sounds better like this:
"If turned on, this light can assist the system administrator in locating one physical machine in the cluster.

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/management/basic/rbeacon.rst [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+Using xCAT, administrators can turn on and off the beacon light using: ``rbeacon <node> on|off`` 
+
+There's currently no way to query whether the beacon light is on or off.  To work around, first turn off all the lights and then turn on the beacon for the machine you wish to identify: ::
```
Comment (gurevichmark) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/management/basic/rbeacon.rst]: I do not quite understand the second sentence. If there is no way to query the light, turning them all off and then one of the lights on is not really a workaround. You still can not query the state, you just have single machine with a beacon on in the cluster.
Perhaps it is best to just leave the first sentence and remove the workaround suggestion ?

**Comment by whowutwut** [comment]
@gurevichmark Thanks for the comments! I've updated the files accordingly.

**Comment by gurevichmark** [comment]
I approve. Can be merged.

**Pull request merged**